### PR TITLE
Initial map graph

### PR DESCRIPTION
**Pull request by wyatt-howe**
- Intitial attempt at creating a page that displays a navigable map of the nodes and edges
- Uses d3 and json, but does not yet reference the actual json files–only an example
[HTML preview](https://htmlpreview.github.io/?https://raw.githubusercontent.com/wyatt-howe/mapofcrypto/master/web/graph/index.html)

### CODE REVIEW

**Comment by sarahscheffler** [comment]
The graph output looks good, but I'm leaving it on its own branch until it pulls from the data.

Requested changes:

1. Let's pull the inputs from source, either by (a) reading the nodes/ and edges/ folders directly, or by writing a script to convert the JSON from nodes/ and edges/ into JSON like you have in links.txt (much like how [web/wiki/BUILD.sh](https://github.com/sarahscheffler/mapofcrypto/blob/master/web/wiki/BUILD.sh) converts the nodes into the format needed by the wiki code).

2. Delete unnecessary comments

**Comment by wyatt-howe** [comment]
Changes made: The graph now pulls data from the main JSON files.
It does so indirectly through a PHP script (can be rewritten as JS if needed) which is run independently to update the graph.
[HTML preview](https://htmlpreview.github.io/?https://raw.githubusercontent.com/wyatt-howe/mapofcrypto/master/web/graph/index.html)

**Comment by wyatt-howe** [comment]
Changes made: The graph now pulls data from the main JSON files in /nodes and /edges.
It does so indirectly through a PHP script (can be rewritten as JS if needed) which is run independently to update the graph.
[HTML preview](https://htmlpreview.github.io/?https://raw.githubusercontent.com/wyatt-howe/mapofcrypto/master/web/graph/index.html)

File: web/graph/index.html [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+html {
+  height: 100%;
+}
+
+body {
+    margin: 0;
+    background-image: url("grid-tile.gif");
+    background-repeat: repeat;
+    background-size: 210px 152px;
+    /* background-color: rgb(245, 252, 255); */
```
Comment (sarahscheffler) [inline web/graph/index.html:16]: There are a few commented out lines in `index.html` that should be removed before we merge.

File: web/graph/links.txt [hunk 2]
```diff
@@ -0,0 +1,56 @@
+[
```
Comment (sarahscheffler) [inline web/graph/links.txt:1]: This file is unnecessary to merge into master.

File: web/graph/primitives.txt [hunk 3]
```diff
@@ -0,0 +1,37 @@
+[
```
Comment (sarahscheffler) [inline web/graph/primitives.txt:1]: This file is unnecessary to merge into master.

**Review by sarahscheffler — COMMENTED** [review]
Awesome, once the unnecessary comments from `index.html` are deleted and the extraneous files removed from the PR, I'll merge.

(Comments themselves are fine, just not the ones that do not provide any useful information.)

**Pull request closed without merging**